framework/util/logging.cpp

```diff
@@ -203,13 +195,16 @@ void Log::LogMessage(
                 // Never add prefixes or indents for the Android console logging.
                 write_prefix_and_indents = false;
 #else  // !__ANDROID__
-                if (output_to_stderr)
+       // If the severity is any diagnostic information other than command logging, send to stderr unless the
+       // user has indicated all diagnostic output should be squashed to stdout.
+                if ((severity == kAlwaysOutputSeverity) || (severity == kCommandSeverity) ||
+                    settings_.output_errors_to_stdout)
                 {
-                    log_file_ptr = stderr;
+                    log_file_ptr = stdout;
                 }
                 else
                 {
-                    log_file_ptr = stdout;
+                    log_file_ptr = stderr;
                 }
```

Comment:
I don't get it either but that's what the code did before.  `kAlwaysSeverity` is used by WRITE_CONSOLE, and that's how `replay` prints out the FPS information that everyone parses, so it needs to continue to go to `stdout`.  You could make an argument that WRITE_CONSOLE maybe shouldn't use LogMessage, but LogMessage does have all the logic in it to write to the debug output on Windows and the logger on Android and a file if specified so it becomes the clearinghouse for output.  `kCommandSeverity` is used by some kind of command logging object that currently isn't used anywhere, but the way the code is structured it would also go to `stdout`, presumably for heavy debugging of layer operation, maybe.  I'm reluctant to change the semantics that are expected.  Changing the default to `stderr` is already causing trouble that I don't want to compound.